crates/subspace-farmer/src/bin/subspace-farmer/commands/farm.rs

```diff
@@ -157,10 +157,11 @@ pub(crate) async fn farm(
         .await
     {
         Ok(ws_server) => ws_server,
-        Err(jsonrpsee::core::Error::Transport(error)) => {
+        Err(jsonrpsee::core::Error::Transport(ref error)) => {
```

Comment:
Hm, didn't need `ref` in Rust for ages, why?

Reply:
I did weird experiments with that. Tracing uses structured logs and it has dedicated type for errors (things which implement `std::error::Error`), but there is no fancy way to send error as error (not as string) to logs.

Will remove it :)